src/components/Poker/GameController/GameController.tsx

```diff
@@ -60,7 +71,16 @@ export const GameController: React.FC<GameControllerProps> = ({ game, currentPla
             titleTypographyProps={{ variant: 'h6' }}
             action={
               <div className='GameControllerCardHeaderAverageContainer'>
-                <Typography variant='subtitle1'>{game.gameStatus}</Typography>
+                <Typography variant='subtitle1' className='GameControllerGameStatus'>
+                  {game.gameStatus}
+                </Typography>
+                {isTimerEnabled ? (
```

Comment:
It will be good to show the timer values to all users in the session too, but this requires changes in the data model. Probably could add as added feature in another PR.